networking/podenv.go

```diff
@@ -277,6 +282,9 @@ func copyFileToDir(src, dstdir string) (string, error) {
 	return dst, err
 }
 
+// Load all network configuration files from the user-supplied configuration
```

Comment:
Comment for function should be started with it's name.
PTAL https://blog.golang.org/godoc-documenting-go-code